packages/lib-classifier/src/store/SubjectStore/SubjectStore.js

```diff
@@ -176,7 +182,18 @@ const SubjectStore = types
       newSubjects.forEach(subject => {
         try {
           const alreadyStored = self.resources.get(subject.id)
-          if (!alreadyStored) {
+          // assume all subjects from Panoptes are unseen by default
+          let notSeen = true
+          // completed workflows fall back to random selection
+          const { user_has_finished_workflow } = subject
+          if (!user_has_finished_workflow && self.prioritized) {
+            const metadataPriority = subject.metadata['#priority'] ?? subject.metadata.priority
+            // subject metadata in the API response are strings, not numbers.
+            const priority = metadataPriority ? parseFloat(metadataPriority) : -1
+            const lastPriority = self.last?.priority || -1
```

Comment:
Existential check for `.last`, solid 👍 I appreciate that we're setting fallback values for these priority valuables.

A small note is that parseFloat(metadataPriority) can still return `NaN`, but the logic still checks out OK for determining the correct value of `notSeen`